domain/entities/errors.go

```diff
@@ -3,6 +3,7 @@ package entities
 import "errors"
 
 var (
+	ErrInvalidMessage     = errors.New("this message does not belong to the topic")
```

Comment:
Could you give a better name to this variable? Like `ErrTopicNameDoesNotMatch`. WDYT @sptGabriel?

Reply:
Yes i go fix linter and name, and other strategy to assert error.